hudi-integ-test/src/test/java/org/apache/hudi/integ/ITTestHoodieDemo.java

```diff
@@ -115,6 +116,7 @@ public void testParquetDemo() throws Exception {
   }
 
   private void setupDemo() throws Exception {
+    Thread.sleep(60 * 1000);
```

Comment:
@nsivabalan : I had earlier suggested this to unblock debugging. Can you see why data-node is coming up occasionally ?

Reply:
I didn't see this in the last 30 odd runs. So, gonna be tough to repro.